src/BlazorUI/Bit.BlazorUI.Extras/Components/AccordionList/BitAccordionList.scss

```
@@ -0,0 +1,9 @@
+@import "../../../Bit.BlazorUI/Styles/functions.scss";
```

Stylelint (17.12.0)

[error] Unexpected extension ".scss" in `@import` (scss/load-partial-extension)

(scss/load-partial-extension)